src/toil/wdl/wdltoil.py

```diff
@@ -622,6 +624,8 @@ def evaluate_output_decls(output_decls: List[WDL.Tree.Decl], all_bindings: WDL.E
     output_bindings: WDL.Env.Bindings[WDL.Value.Base] = WDL.Env.Bindings()
     for output_decl in output_decls:
         output_value = evaluate_decl(output_decl, all_bindings, standard_library)
+        drop_if_missing_with_workdir = partial(drop_if_missing, work_dir=getattr(standard_library, "_execution_dir"))
```

Comment:
It would probably be better if all standard_library's were typed as `ToilWDLStdLibBase`; this was mainly done for mypy as mypy thinks the type is `WDL.StdLib.Base` and not `ToilWDLStdLibBase`.

I can probably turn the gettr into a method, although this current logic may not be sufficient. Another issue I found is that this will coerce all nonexistent files to null, even if the specified type is not optional. For example:
```wdl
...
output {
  File f = "nonexistent.txt"
}
...
```

This wasn't an issue previously as we ensured file existence at type coercion.

Reply:
This actually refers to https://github.com/DataBiosphere/toil/issues/5006